roles/openshift_clock/tasks/main.yaml

```diff
@@ -12,6 +12,11 @@
   register: result
   until: result is succeeded
 
+- name: detect network time status
+  shell: "timedatectl | grep -oP '(?<=NTP synchronized: )(yes|no)'"
```

Comment:
Would it be better to check NTP enabled rather than synchronized?

Reply:
Right, it seems there is no universal way to do this - services responsible for NTP are different in Fedora and CentOs. Even the output of `timedatectl` is different - hence PAPR bot failures.

This sounds like too much work for a small change, I'll close this PR unless some universal solution pops up

Reply:
We could check using different logic on Fedora and RHEL/Centos but I'm also fine with foregoing this. I'd hope that timedatectl is idempotent.

Reply:
`timedatectl`'s output shouldn't be considered stable (but `timedatectl set-ntp` exists in all systems). Closing this as its just not worth it